src/api/models/article.js

```diff
@@ -0,0 +1,57 @@
+import slug from 'slug';
+
+const THIRTY_SIX = 36;
+const SIX = 6;
+const ZERO = 0;
+
```

Comment:
use described variable name

Reply:
Can you explain what this mean

Reply:
@murediane variable names should be meaningful and descriptive so that others reading our code understand the context behind the variable.

Reply:
There is no meaning for the variable apart from it being a number . I was avoiding a no magical  number error .because I am just using those number to get the help to generate the slug but they have no specific role that can be understood.they are just numbers